source/adminguide/storage.rst

```diff
@@ -237,6 +238,66 @@ templates, and ISOs.
 .. note::
    If files or folders are not associated with a cloudstack resource, it doesn't mean that they are not used by cloudstack.
 
+
+Setting NFS Mount Options on the Storage Pool
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NFS mount options can be added while creating an NFS storage pool for
+KVM hosts. When the storage pool is mounted on the KVM hypervisor host,
+these options will be used. Options currently tested and supported are
+`vers` and `nconnect`. 
+
+Although it depends on the NFS server, but commonly supported `vers` values 
+are `3` for NFSv3 and minor versions `4.0, 4.1 and 4.2` for NFSv4. 
+`nconnect` values can range from 1 to 16.
+
+Administrator can give the NFS mount options while adding a Primary Storage
+from the Create Zone Wizard as well as the Add Primary Storage form.
+|nfs-mount-options-create-zone-wizard.png|
+|nfs-mount-options-add-primary-storage.png|
+
+NFS mount options can be changed on a pre-existing Storage Pool using the
+Edit Primary Storage form. Storage Pool needs to be unmounted and mounted
+again on the KVM host using the new options. Running VMs using volumes
+in the Storage Pool either need to be stopped or the volumes migrated.
+So the NFS Mount Options field below is only shown if the Storage Pool
```

Comment:
I think, this and below 2 lines may not be required after updated with above suggestions